ember/app/styles/components/_not-found.scss

```diff
@@ -0,0 +1,8 @@
+.not-found {
+  display: grid;
+  place-items: center;
+
+  h1 {
+    @extend .uk-heading-2xlarge;
+  }
+}
```

Comment:
Let's use the uikit classes directly to avoid unnecessary CSS.

Reply:
done here https://github.com/adfinis/Outdated/pull/72/commits/97a99b16d5b65943b98f21f70f841b1096ff6202